docs/src/installation.md

```diff
@@ -58,6 +58,9 @@ Where:
 -   SDP = Semidefinite programming
 -   MISDP = Mixed-integer semidefinite programming
 
+You may also use [AmplNLWriter](https://github.com/JuliaOpt/AmplNLWriter.jl) to
+access solvers that support the [nl format](https://en.wikipedia.org/wiki/Nl_(format)).
```

Comment:
I would add "such as [Bonmin](https://projects.coin-or.org/Bonmin) and	
  [Couenne](https://projects.coin-or.org/Couenne)". A user might do Ctrl+F here to try to see if Bonmin or Couenne is supported.